kevm-pyk/src/kevm_pyk/kompile.py

```diff
@@ -64,7 +63,7 @@ def kevm_kompile(
     if llvm_library is None:
         llvm_library = output_dir / 'llvm-library'
 
-    if target in [KompileTarget.LLVM, KompileTarget.HASKELL_BOOSTER] and not plugin_dir:
+    if target == KompileTarget.LLVM and not plugin_dir:
```

Comment:
`plugin_dir` should now become a mandatory parameter, and checks and assertions that it is not `None` removed.

Reply:
Done.